source/administration/upgrade.md

```diff
@@ -4,48 +4,51 @@ This guide explains how to upgrade your Mattermost deployment across versions an
-             - If you have more than one team, specify the `primary team` based on the team you use the most.
-                  - If your server contains duplicate accounts (multiple accounts with either the same email addresses or the same usernames) the user account in the `primary team` will be considered the primary account and remain unchanged.
-                     - When the server finds a duplicate account not in the `primary team` the email address of the account may be changed to avoid conflicts
-                         - An account with a **duplicate email address** will be updated so `+[TEAM_URL_NAME]` is appended to the local part of the email address. For example: An account with a duplicate email `steve@company.com` in the team at URL `https://mattermost.company.com/marketing` becomes `steve+marketing@company.com`. The `+marketing` used in this procedure is part of the RFC5233 email specification and most email systems will properly route `steve+marketing@company.com` to `steve@company.com`. After the upgrade, if email authentication is used for sign-in, the user would need to sign-in with the new email address.
-                         - An account with a **duplicate username** will be updated so `[TEAM_URL_NAME].`” is prepended to the username. For example: An account with a duplicate username `steve` in the team at URL `https://mattermost.company.com/marketing` becomes `marketing.steve`.
-         - Users with accounts containing duplicate emails or usernames will receive a notification email explaining the upgrade, and instructions on how to move to a single user account ([see example](http://www.mattermost.org/upgrading-to-mattermost-3-0/#notification))
+             - Enter the most used team, or the only team, on your server.
+                  - If your server contains duplicate accounts (multiple accounts with either the same email addresses or the same usernames), see [duplicate accounts](http://docs.mattermost.com/administration/upgrade.html#duplicate-accounts).
 7. Start your server and address any setting changes relevant in the latest version of Mattermost
       1. Run `sudo start mattermost`.
       2. Opening the **System Console** and saving a change will upgrade your `config.json` schema to the latest version using default values for any new settings added. 
 8. Test the system is working by going to the URL of an existing team.
       1. You may need to refresh your Mattermost browser page in order to get the latest updates from the upgrade.
-9. After the Mattermost 3.0 upgrade users with duplicate accounts can follow instructions in the upgrade email they received to login to teams on which the duplicate accounts were created and add their primary account to the team and any private groups that are still actively used. Users can continue to access the direct message history of their duplicate accounts using their updated email addresses.
+
+#### Duplicate Accounts
```

Comment:
@DavidLu1997 not sure if this section is rendering as you intended? 

![image](https://cloud.githubusercontent.com/assets/177788/15439508/f1727f68-1e85-11e6-90d9-7d1c84866b7a.png)